src/components/panel/panel.js

```diff
@@ -489,6 +482,16 @@ angular
 
 /**
  * @ngdoc method
+ * @name MdPanelPosition#tryPanelPositions
+ * @description
+ * Specify an ordered list of panel positions. The first position which
+ * allows the panel to be completely on-screen will be chosen; the last position
+ * will be chose whether it is on-screen or not.
```

Comment:
Can you show an example of how this is used? It will be helpful in explaining the functionality of this method.